buildSrc/src/main/kotlin/CommonJavaConfig.kt

```diff
@@ -67,6 +67,7 @@ fun Project.applyCommonJavaConfiguration(sourcesJar: Boolean, banSlf4j: Boolean
                     "https://jd.papermc.io/paper/1.18/",
                     "https://intellectualsites.github.io/fastasyncworldedit-javadocs/worldedit-core/"
             )
+            docTitle = "${rootProject.name}-${project.description}" +  " " + "${rootProject.version}"
```

Comment:
Is there a reason why we're mixing string interpolation and string concatenation?

Reply:
That's the same format used everywhere else, e.g. see the pom generator.

Reply:
I see. I guess it would make sense to clean that up at some point, but that can be done separately